spec.md

```diff
@@ -1076,6 +1076,36 @@ $instance = new class extends \Foo implements
 };
 ~~~
 
+## 9. Enumerations
+
+Enumerations MUST follow the same guidelines as classes, except where otherwise noted here.
+
+Methods in Enumerations MUST follow the same guidelines as methods in classes.  Non-public methods MUST use `private`
```

Comment:
I'm thinking we also adjust the term defined in section 4 to include enums since they follow the same rules.